gremlin/values.yaml

```diff
@@ -46,11 +46,15 @@ gremlin:
   # NOTE: this is disabled by default to maintain behavior of previous versions
   hostPID: false
 
-  # gremlin.hostPID -
+  # gremlin.hostNetwork -
   # This must be true for any Gremlin installation that wishes to carry out network attacks at the host-level
   #   (e.g. impact network traffic to all host processes and containers)
   hostNetwork: false
 
+  # gremlin.installApparmorProfile -
+  # This controls if gremlin installs it's own apparmor profile.  This is only necessary if you're running apparmor
+  installApparmorProfile: false
```

Comment:
We probably want to put this under the `podSecurity` object, analogous to SELinux and Seccomp. Something like

```
--set gremlin.podSecurity.apparmor.enabled=true
```

Reply:
`apparmor.enabled` would be incorrect because we don't enable apparmor, but more to the point the feature here is to deploy out a thing which installs apparmor profiles.  We incidentally configure to the profile we install, but it is possible to still override that configuration via:

```
--set gremlin.installApparmorProfile=true
--set gremlin.apparmor="localhost/someotherprofile"
```

The later line is enabling apparmor enforcement where as the former is just installing the profile

We can also discuss migrating the `gremlin.apparmor` flag, but maybe that should be a separate change?